co2eq/digital/email.js

```diff
@@ -0,0 +1,33 @@
+import { ACTIVITY_TYPE_DIGITAL, DIGITAL_CATEGORY_EMAIL } from '../../definitions';
```

Comment:
Let's remove this carbon model for now as we don't currently have the ability to add a new activity type (due to UI constraints).
Let's however make sure we keep reference of the carbon model so we quickly can add it when we have the ability to support it.

Reply:
So comment it all out?

Reply:
I'd remove the whole file from this PR and maybe put it as a gist so we don't loose it?

Reply:
https://gist.github.com/baywet/38f21c202db5baf22a630ccbb7bae2ef